consensus/ethash/algorithm.go

```diff
@@ -39,30 +39,47 @@ const (
 	datasetGrowthBytes = 1 << 23 // Dataset growth per epoch
 	cacheInitBytes     = 1 << 24 // Bytes in cache at genesis
 	cacheGrowthBytes   = 1 << 17 // Cache growth per epoch
-	epochLength        = 30000   // Blocks per epoch
+	oldEpochLength     = 30000   // Blocks per epoch pre ECIP-1099 activation
+	newEpochLength     = 60000   // Blocks per epoch post ECIP-1099 activation
```

Comment:
`old` and `newEpochLength` at least need to be renamed to explain their association to ECIP1099, since this change is only old/new for Ethereum Classic and her testnets.  Maybe `epochLengthAppendixJ` and `epochLengthECIP1099`? 

Got the `AppendixJ` from the [yellow paper](https://ethereum.github.io/yellowpaper/paper.pdf).

Reply:
I would lean towards `epochLengthDefault` as its more descriptive at first glance. AppendixJ, altho from the paper, is a little obscure.

Reply:
Also the Yellow Paper gets edited from time to time, so Appendix J may not be an ultimate reference either.